site/content/docs/user/known-issues.md

```diff
@@ -33,7 +33,6 @@ description: |-
 * [Pod Errors Due to "too many open files"](#pod-errors-due-to-too-many-open-files) (likely [inotify] limits which are not namespaced)
 * [Docker Permission Denied](#docker-permission-denied) (ensure you have permission to use docker)
 * [Windows Containers](#windows-containers) (unsupported / infeasible)
-* [Non-AMD64 Architectures](#nonamd64-architectures) (images not pre-built yet)
```

Comment:
Non-AMD64 and ARM64

Reply:
I wonder if "Unsupported Architectures" might be better than explicitly listing out the supported ones. That could keep it flexible in the unlikely event more than AMD64 and ARM64 are ever supported.

Reply:
I like that idea

Reply:
I just pushed a new commit with this suggestion. Let's hope the 3rd time is the charm :-)